Calling out Sender allow corner case (#8451)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/manage-tenant-allows.md

>
> - Based on what filters determined the mail to be malicious, during mailflow, the allows are added. For example, if filters found both sender and URL to be bad, an allow will be added for each. 

> > - When that entity (sender, domain, URL, file) encountered again, all filters associated with that entity are skipped.

Reviewer: ```suggestion
> - When that entity (sender, domain, URL, file) is encountered again, all filters associated with that entity are skipped.
```

> > - So for an email (containing this entity), during mailflow, if the rest of the filters find the email to be clean then the email will be delivered. So for example, sender allow (when authentication passes) will bypass all verdict except for malware and HCP associated with an attachment or URL.

Reviewer: ```suggestion
> - So for an email (containing this entity), during mail flow, if the rest of the filters find the email to be clean then the email will be delivered. So for example, sender allow (when authentication passes) will bypass all verdicts except for malware and HCP associated with an attachment or URL.
```

> - Only the _combination_ of the spoofed user _and_ the sending infrastructure as defined in the domain pair is specifically allowed or blocked from spoofing.
> - When you configure an allow or block entry for a domain pair, messages from that domain pair no longer appear in the spoof intelligence insight.
> - Entries for spoofed senders never expire.

> > - Spoof supports both allow and block through . URL supports only block.

Reviewer: ```suggestion
> - Spoof supports both allow and block through. URL supports only block.
```
